app/assets/javascripts/views/settings_view.js.coffee

```diff
@@ -18,7 +18,6 @@ class ELMO.Views.SettingsView extends ELMO.Views.ApplicationView
     return false
 
   show_change_credential_fields: (event) ->
-    @$(event.target).hide()
```

Comment:
I don't think this is going to work Jay. This thing is supposed to hide the 'Show' link after it is clicked and the fields are shown.

The way to fix this is to make the `'click .adapter-settings a'` more specific so that it only listens to the intended link

(like this one)

![image](https://user-images.githubusercontent.com/702320/44471184-5f5a6700-a5f9-11e8-96ad-8f7936b996ac.png)

and does _not_ listen to the hint link.

Lesson: If you are removing code like this you need to have a good reason for doing so. If your assumption was that this code did nothing other than cause a bug, you would need to be sure of that. That would involve:

1. Trying to figure out what the code was intended to do by tracing through it.
2. If you still don't know, asking the tech lead.
3. If the tech lead says it's fine to delete, then you're good to go.